tests/test_plugin.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from poetry_partifact_plugin.plugin import PartifactPlugin
+
+
+@pytest.fixture
```

Comment:
nit: Put these in a separate `conftest.py` file so it's easier to grok the test cases in this file?